clr-k8s-examples/README.md

```diff
@@ -7,8 +7,28 @@ This setup currently will work with k8s 1.14 & above. Any version of k8s before
 
 To be able to test this tool, you can create a 3-node vagrant setup. In this tutorial, we will talk about using [libvirt](https://github.com/vagrant-libvirt/vagrant-libvirt), but you can use any hypervisor that you are familiar with.
 
-* Install vagrant on the distro you are using. Steps can be found at [Vagrant docs](https://www.vagrantup.com/intro/getting-started/install.html#installing-vagrant)
-* `vagrant up --provider=libvirt`
+## Install vagrant on Ubuntu
+
+On Ubuntu Bionic, run these commands
+Install dependencies and prepare system
+```bash
+sudo apt-get update
+sudo apt-get install gcc make
```

Comment:
why gcc and make?

Reply:
They are required during `vagrant plugin install vagrant-libvirt', otherwise an error is shown and initially the logs complain of missing gcc.  Make is also used during the plugin install process.